descwl_coadd_task/tests/test_warps.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import annotations
+
+import unittest
+
+import lsst.afw.cameraGeom.testUtils
+import lsst.afw.image as afw_image
+import lsst.skymap as skyMap
+import lsst.utils.tests
+import numpy as np
+from lsst.afw.detection import GaussianPsf
+from lsst.daf.butler import DataCoordinate, DimensionUniverse
+from lsst.pipe.base import InMemoryDatasetHandle
+from lsst.pipe.tasks.coaddBase import makeSkyInfo
+
+from descwl_coadd_task import MakeShearWarpConfig, MakeShearWarpTask
+
+PIXEL_SCALE = 0.2
+"""Pixel scale in arcseconds"""
+
+
+class MakeWarpTestCase(lsst.utils.tests.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Docstring inherited.
+        # This method contains the setup that needs to be done only once for
+        # all the tests.
+        cls.ny = 4000
+        cls.nx = 4072
+
+        cls._build_skyMap()
+        cls._generate_photoCalib()
+        cls.skyInfo = makeSkyInfo(cls.skyMap, 0, 36)
+
+        # Setup the metadata and empty exposures for warping.
+        cls._make_data()
```

Comment:
Don't the data get modified in place?  In that case don't we want to remake the data for each unit test?

Reply:
This doesn't set the pixel values. It just sets up the geometry, which can be the same for all the tests.